test/Event/EventTest.php

```diff
@@ -125,6 +126,38 @@ public function it_throws_an_error_when_creating_an_event_with_a_non_string_even
         );
     }
 
+    /**
+     * @test
+     */
+    public function it_sets_the_audience_type_to_education_when_creating_an_event_with_a_dummy_education_location()
+    {
+        $locationUuid = UUID::generateAsString();
```

Comment:
I've been using `$dummyPlaceId` in tests. Should we aim for some consistency?

Reply:
In the context of events we generally use location (historically). As in: "the event's location is this particular place".

Reply:
That makes sense. I'll try to keep the wording consistent with that rule.